nixos/modules/programs/clash-verge.nix

```diff
@@ -1,9 +1,14 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 
 {
   options.programs.clash-verge = {
     enable = lib.mkEnableOption "Clash Verge";
-    package = lib.mkPackageOption pkgs "clash-verge" {};
+    package = lib.mkPackageOption pkgs "clash-verge-rev" { };
```

Comment:
I think Aleksana recommended to use throw by default

cc @Aleksanaa

Reply:
I don't mean here, Uh, good question actually.